SayTheirNames/Source/View/ImageWithBlurView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ImageWithBlurView.swift
+//  SayTheirNames
+//
+//  Created by Leonard Chen on 6/5/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class ImageWithBlurView: UIView {
+    // MARK: - View
+    lazy var visualEffectView: UIVisualEffectView = {
+        let effect = UIBlurEffect(style: .dark)
+        let view = UIVisualEffectView(effect: effect)
+        view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        return view
+    }()
```

Comment:
Let's try not to expose too much implementation details. 
- Keep properties private 
- Use `let` over `var` when we can
```
 private let visualEffectBlur = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
```